### PR TITLE
Controller and model fixes

### PR DESCRIPTION
**Pull request by Artyom-Borushko**

### CODE REVIEW

File: src/controller/infoPanelController.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import {
+  gameScreen, endGameScreen, endGameScreenHeader, ammoProgressBar, countDownTimer,
+  remainingAmmo as ammoInfo,
+} from '../constants/querySelectors';
+import { updateUserLocalStorageTime, removeUserFromLocalStorage, updateUserLocalStorageAmmo } from './localStorageController';
+import { generateFireIconForInfoPanel, getSunkBoatImageLocator, calculateMinutesAndSeconds } from '../model/infoPanelModel';
+
+let availableAmmo = 30;
+let defaultAmmo = 0;
+// eslint-disable-next-line import/no-mutable-exports
```
Comment (katrin2404) [inline src/controller/infoPanelController.js]: Can you avoid linter rule disabling?

File: src/model/battleshipGridModel.js [hunk 2]
```diff
@@ -0,0 +1,169 @@
+/* eslint-disable no-param-reassign */
```
Comment (katrin2404) [inline src/model/battleshipGridModel.js]: Avoid disabling of rule, please.

File: src/controller/infoPanelController.js (hunk 1 above)
Comment (Artyom-Borushko) [reply src/controller/infoPanelController.js]: fixed with commit - a3d0bec

File: src/model/battleshipGridModel.js (hunk 2 above)
Comment (Artyom-Borushko) [reply src/model/battleshipGridModel.js]: fixed with commit - a1dbeb1

**Pull request merged**